include/mesh/boundary_info.h

```diff
@@ -912,6 +923,8 @@ class BoundaryInfo : public ParallelObject
    * See _side_boundary_ids, _edge_boundary_ids, _node_boundary_ids, and
    * _shellface_boundary_ids for sets containing IDs for only sides, edges,
    * nodes, and shell faces, respectively.
+   *
+   * This data is semilocal in nature
```

Comment:
I lazily didn't check, but is the concept of "semilocal data" rigorously defined somewhere?

Reply:
I actually don't think so, and I don't know where to put it. Instead I could say "attached to this process's local or ghosted elements"